### PR TITLE
Rspec exceptions

### PR DESCRIPTION
**Pull request by smoroso**

### CODE REVIEW

File: rubocop.yml [hunk 1]
```diff
@@ -106,3 +108,13 @@ Style/SignalException:
 
 Layout/SpaceAroundOperators:
   AllowForAlignment: true
+
+Metrics/ModuleLength:
+  Exclude:
+    - "**/*_spec.rb"
+
+RSpec/MultipleExpectations:
+  Max: 10
```
Comment (fanaugen) [inline rubocop.yml]: I think if you allow 10 expectations, you might as well disable this cop completely (no limit).

File: rubocop.yml [hunk 2]
```diff
@@ -106,3 +108,13 @@ Style/SignalException:
...
+
+RSpec/NestedGroups:
+  Max: 6
```
Comment (fanaugen) [inline rubocop.yml]: Same here: might as well turn it off: if anyone tries to write a spec with seven levels of nesting, it should look suspicious (like a symptom of a possible code smell) either while writing or in code review

File: rubocop.yml [hunk 3]
```diff
@@ -106,3 +108,13 @@ Style/SignalException:
 
 Layout/SpaceAroundOperators:
   AllowForAlignment: true
+
+Metrics/ModuleLength:
+  Exclude:
+    - "**/*_spec.rb"
```
Comment (fanaugen) [inline rubocop.yml]: 👍

File: rubocop.yml [hunk 4]
```diff
@@ -59,6 +59,8 @@ Metrics/MethodLength:
 
 Metrics/ParameterLists:
   Max: 4
+  Exclude:
+    - "**/*_spec.rb"
```
Comment (fanaugen) [inline rubocop.yml]: hmm, what about refactoring that spec helper method to accept an `options` hash instead? Then you can have as many parameters as you want, and the keys describe their semantics. 

In this particular case I happen to side with the cop; I would even say when it comes to positional arguments, the limit four is not hard enough: when you’re not looking at a method with four arguments, you’re almost guaranteed to forget which is which

**Review by fanaugen — CHANGES_REQUESTED** [review]

File: rubocop.yml (hunk 2 above)
Comment (smoroso) [reply rubocop.yml]: Turning it off is fine for me

File: rubocop.yml (hunk 4 above)
Comment (smoroso) [reply rubocop.yml]: It's actually an explicit choice I made to not use an option object.

As an example, please check the current file in loveos-backend and especially the `atttributes_expectations` method: spec/api/containers/checkouts/seller_calculation_spec.rb

It allows me to test my calculation service in one line by just changing numbers in the params. Using an option object would not provide me any benefit from using the `attributes_expectation` method

Comment (fanaugen) [reply rubocop.yml]: hmm, I guess I see your point… that method looks like it wants to be a `shared_examples` block, but that, too, would be more verbose

**Review by fanaugen — APPROVED** [review]
kill the annoying cops!

**Pull request merged**